apis/metal3.io/v1alpha1/bmceventsubscription_validation.go

```diff
@@ -29,23 +28,22 @@ var bmclog = logf.Log.WithName("bmceventsubscription-validation")
 
 // validateSubscription validates BMCEventSubscription resource for creation
 func (s *BMCEventSubscription) validateSubscription() []error {
-	bmclog.Info("validate create", "name", s.Name)
 	var errs []error
 
 	if s.Spec.HostName == "" {
-		errs = append(errs, fmt.Errorf("HostName cannot be empty"))
+		errs = append(errs, errors.New("hostName cannot be empty"))
```

Comment:
We should be validating this client-side, you don't need a webhook just to make sure a field is not empty.

Reply:
I agree, but I'd rather keep this PR refactoring-only